src/main/js/samples.js

```diff
@@ -71,7 +71,12 @@ samples.push({
         "   // Mark the code build 'stage'....\n" +
         "   stage 'Build'\n" +
         "   // Run the maven build\n" +
-        "   sh \"${mvnHome}/bin/mvn -Dmaven.test.failure.ignore clean package\"\n" +
```

Comment:
Only worked on Unix, and failed to account for the possibility of a workspace path with spaces.